Lib/test/test_memoryview.py

```diff
@@ -679,5 +681,24 @@ def test_picklebuffer_reference_loop(self):
         self.assertIsNone(wr())
 
 
+@threading_helper.requires_working_threading()
```

Comment:
I'm not totally sure if it's possible for `requires_resource("cpu")` to fail while `requires_working_threading()` passes, but we might as well be safe:
```suggestion
@threading_helper.requires_working_threading()
@support.requires_resource("cpu")
```

Reply:
Hi @ZeroIntensity ,

Thanks very much for your suggestion! 😊

I find that case failed on wasm architecture.

Does it relate to this requirement?

Best Regards,
Edward